### ISSUE TITLE
Infinite reload while relogin dialog is shown

### ISSUE THREAD

**Issue by markusguenther** [neos/neos-ui#2590]
Was reported by @creative-resort in the neos distribution.

### Description

The re-login form is re-loaded in an endless loop

### Steps to Reproduce

1. Log in to a demo site and click on a page, e.g. "Features" in In-Place Edit Mode
2. Let the session's login duration expire.
3. In the navigation tree, click on a different page (e.g. "Shortcuts"), so a re-login is initiated.

#### Expected behavior

Normal re-login screen.

#### Actual behavior

Here is a video of how the re-login form keeps being reloaded in a loop:

![neos login error](https://user-images.githubusercontent.com/12367437/69498750-aa91b280-0eeb-11ea-989b-5281452b35d1.gif)

As the form is constantly re-loaded, logging in is impossible.
Only after canceling the page loading via browser button "cancel", the reloading stops.

### Affected Versions

Neos: 5.0.4
Flow: 6.0.5

PHP 7.3

Browser: latest Google Chrome Browser

**Comment by markusguenther** [comment]
Thought we had something like that in the past, but did not find the issue yet. So need to investigate that later. As we have feature freeze on 1st of december will wait with bugfixes until the freeze ;)

**Comment by Sebobo** [comment]
Is this still happening?

I cannot reproduce it currently.

**Comment by grebaldi** [comment]
As per this description:
> The re-login form is re-loaded in an endless loop

This is not happening anymore.

However, there still is a slight problem: The loading indicator runs on a loop when the Login Dialog appears. Similar behavior has been described elsewhere:

- https://github.com/neos/neos-ui/issues/1718
- https://github.com/neos/neos-ui/issues/1664

I think it's best to have a separate issue to address the loading indicator behavior.

I'll close this one.

**Comment by durimjusaj** [comment]
This issue still occurs in Neos 9.1.5.
